src/app/authentication/service/signInService.go

```diff
@@ -0,0 +1,45 @@
+package service
+
+import (
+	"Stock_broker_application/constants"
+	"Stock_broker_application/models"
+	"Stock_broker_application/repo"
+)
+
+// SignInService handles the sign-in logic
+type SignInService struct {
+	UserRepository repo.UserRepository
```

Comment:
don't capitalize the variable if it isn't used anywhere.
rename it to userRepository